consensus/polybft/event_tracker.go

```diff
@@ -0,0 +1,79 @@
+package polybft
+
+import (
+	"context"
+	"path/filepath"
+
+	hcf "github.com/hashicorp/go-hclog"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/jsonrpc"
+	"github.com/umbracle/ethgo/tracker"
+	boltdbStore "github.com/umbracle/ethgo/tracker/store/boltdb"
+)
+
+type eventSubscription interface {
+	AddLog(log *ethgo.Log)
+}
+
+type eventTracker struct {
+	dataDir    string
+	config     *PolyBFTConfig
+	subscriber eventSubscription
+	logger     hcf.Logger
+}
+
+func (e *eventTracker) start() error {
+	provider, err := jsonrpc.NewClient(e.config.Bridge.JSONRPCEndpoint)
+	if err != nil {
+		return err
+	}
+
+	store, err := boltdbStore.New(filepath.Join(e.dataDir, "/deposit.db"))
+
+	if err != nil {
+		return err
+	}
+
+	e.logger.Info("Start tracking events", "bridge", e.config.Bridge.BridgeAddr)
+
+	tt, err := tracker.NewTracker(provider.Eth(),
+		tracker.WithBatchSize(10),
+		tracker.WithStore(store),
+		tracker.WithFilter(&tracker.FilterConfig{
+			Async: false,
+			Address: []ethgo.Address{
+				ethgo.Address(e.config.Bridge.BridgeAddr),
+			},
+		}),
+	)
+	if err != nil {
+		return err
+	}
+
+	go func() {
```

Comment:
Could we create issues for such cases when we create goroutines with no ownership and no way to stop them is needed?